src/UniversalRewardsDistributor.sol

```diff
@@ -192,7 +189,9 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
         root = newRoot;
         ipfsHash = newIpfsHash;
 
-        delete pendingRoot;
+        if (pendingRoot.submittedAt != 0) {
+            _setPendingRoot(PendingRoot(0, 0, 0));
+        }
 
         emit EventsLib.RootSet(newRoot, newIpfsHash);
```

Comment:
Can we have this event emitted right after the `root` and `ipfsHash` assignment? Before `_setPendingRoot`

Reply:
outdated

Reply:
outdated